csharp/Platform.Numbers/CalcResult.cs

```diff
@@ -0,0 +1,461 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+
+namespace Platform.Numbers
+{
+    public class CalcResult<T>
```

Comment:
Class will be allocated in the heap and will be tracked by Garbage Collector. The struct will use less resources.

Reply:
Yeah, I'll bear that in mind.